install/kubernetes/Makefile

```diff
@@ -28,6 +28,7 @@ EXPERIMENTAL_INSTALL := "$(ROOT_DIR)/$(RELATIVE_DIR)/experimental-install.yaml"
 CILIUM_CHARTS := "$(ROOT_DIR)/$(RELATIVE_DIR)/cilium"
 CILIUM_VALUES := "$(CILIUM_CHARTS)/values.yaml"
 CHART_FILE := "$(CILIUM_CHARTS)/Chart.yaml"
+USE_DIGESTS ?= false
```

Comment:
Does this get set `true` somewhere?

The way I set this up in the older branches is to set it true by default in this makefile, then *only* when running the `start-release.sh`, we pass `false` directly in. So when preparing a release tag, it will be set to false but for the rest of the time (including the followup commits when creating the helm charts) it will be set true. Does that workflow work?

Reply:
One thing to note with the above workflow is that the smoke tests for generating `quick-install.yaml` will fail for the tag commit, since `USE_DIGESTS` would actually be true during the CI generation of those YAML files (which is different from the committed files which were generated with `USE_DIGESTS=false`. But then once you prepare the followup commit with the actual digests, the smoke tests would pass again.